src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a separate variable for the radius and pass it on for better code readability.